docs/modules/community/pages/release.adoc

```diff
@@ -15,3 +15,137 @@ Don't forget to add your key to https://downloads.apache.org/james/KEYS
 
     $ ssh people.apache.org
     $ cd /www/www.apache.org/dist/james
+
+== Release cheat sheet
+
+First, discuss your intention to perform a release on server-dev mailing list, to ensure a consensus would be reached
+before starting this time-consuming process.
+
+Then you can prepare the release materials:
+
+ * Ensure the `CHANGELOG.md` and `upgrade-instructions.md` files are up to date.
+ * Freeze your version in it and initiate a new iteration.
+ * Write a blog post within `/src/homepage/_post` about your release.
+ * Update the documentation website `/docs`, `src/site`, and `README.md` to point to your new release.
+ * Update the `/dockerfiles`
+
+Open a pull request to validate the changes.
+
+Once merged, it is time for the maven release.
+
+Set up your maven configuration in `~/.m2/settings.xml` :
+
+[source,bash]
+----
+<settings>
+  <servers>
+    <!-- To publish a snapshot of some part of Maven -->
```

Comment:
> To publish a snapshot of some part of Maven

I think you meant james ... or did I misunderstand the comment
also do we need auth for snapshot publication ? I though this was automated on CI ...

Reply:
Not deploying leads to a failing mvn release - what happened to me on friday.

This step is mentionned in the Apache wide fundation, hence we should have it here.

Reply:
BTW I can remove this comment.

Reply:
thanks, I am unfamiliar with push apache releases :D